drivers/cyberpower-mib.c

```diff
@@ -44,6 +43,10 @@ static info_lkp_t cyberpower_power_status[] = {
 	{ 9, "OL BYPASS", NULL, NULL },	/* onBypass */
 	{ 10, "OL TRIM", NULL, NULL },	/* onBuck */
 	{ 11, "OL OVER", NULL, NULL },	/* onOverload */
+
+	/* Note: a "NULL" string must be last due to snmp-ups.c parser logic */
+	{ 1, "NULL", NULL, NULL },	/* unknown */
```

Comment:
Should this just be "" like it is below in cyberpower_battery_status? If the value returned being checked is 1 it will stop iteration without adding "NULL" to the status line, though in practice it's the same result whether it's "" or "NULL".

Reply:
Sorry, can't really say and can't investigate deep at the moment.

In original git commit from a decade ago that introduced this table, it was already like this. Later https://github.com/networkupstools/nut/commit/2438ff5e7ae2dcc505934add1b1bfc2cb0ab51f8 changed some `"NULL"` strings to known states, leaving this one in the middle of remaining table.